.circleci/config.yml

```diff
@@ -68,8 +68,10 @@ jobs:
       - restore_cache:
           key: v4-lint
       - run:
-          name: Install jshint
-          command: sudo npm install -g jshint
+          name: Install eslint and its dependencies
+          command: |
+            sudo npm install -g eslint
+            npm info eslint-config-airbnb-base@latest peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs sudo npm install --save-dev eslint-config-airbnb-base@latest
```

Comment:
Why? :)

Reply:
Using the JavaScript style guide from AirBnb depends on some packages, it's described [here](https://www.npmjs.com/package/eslint-config-airbnb-base).

I know it's ugly, but if we had npm5+, we could use something a lot cleaner.

Reply:
Those are our containers. We can have whatever we want :-)

Reply:
I will need to change the container to have a newer version of `npm`, hopefully it's new enough in Leap 42.3

Reply:
I guess not but there is always [devel:languages:nodejs](https://build.opensuse.org/package/show/devel:languages:nodejs/nodejs10) :cowboy_hat_face: